implementations/rust/ockam/ockam_api/src/kafka/portal_worker.rs

```diff
@@ -682,10 +686,11 @@ mod test {
     }
 
     #[allow(non_snake_case)]
-    #[ockam_macros::test(timeout = 5_000)]
+    #[ockam_macros::test(timeout = 500_000)]
```

Comment:
Is their a particular reason to increase the timeout that much?

Reply:
Left-over from testing. Removed.